portfolio/src/main/webapp/AboutMe.html

```diff
@@ -6,9 +6,9 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body style="background-color:#FEFBE9;">
```

Comment:
I'd try to keep styles in your style.css file. Then you could just apply to all `body` tags in one place.

```
body {
  background-color: #fefbe9;
}
```